integration_test/third_party_apps_data/applications/vault/metadata.yaml

```diff
@@ -17,6 +17,48 @@ short_name: Vault
 long_name: Hashicorp Vault
 description: |-
   Vault is an identity-based secrets and encryption management system. This integration collects Vault's audit logs. The integration also collects token, memory, and storage metrics.
+configure_integration: |-
```

Comment:
@mtabasko i wonder if some content within this `configure_integration` have things that will not agree with the google doc internal markdown. We shall see if it works out or not when we run the doc generator after ops-agent's next release

Reply:
I don't see anything obviously problematic, but I agree that we won't know for sure until we run the generator.